cypress/e2e/agreements/agreement-notes/create-agreement-notes.cy.js

```diff
@@ -40,7 +30,7 @@ describe('Agreement Notes', () => {
   });
 
   it('C1308 Create a note for an Agreement record', () => {
```

Comment:
should have team name at the end of test name - 'C1308 Create a note for an Agreement record (erm)'